pkgs/games/osu-lazer/default.nix

```diff
@@ -77,7 +77,7 @@ buildDotnetModule rec {
   })];
 
   meta = with lib; {
-    description = "Rhythm is just a *click* away";
+    description = "Rhythm is just a *click* away (no score submission or multiplayer)";
```

Comment:
Can you make this change in a separate commit?

Reply:
removed in 411cdf44e0184556543183c697f8abb1184aa73d